test/default-xpring-client-test.ts

```diff
@@ -436,17 +454,52 @@ describe('Default Xpring Client', function(): void {
     assert.equal(exists, true)
   })
 
-  it('Check if account exists - failing network request', async function() {
-    // GIVEN a XpringClient which wraps an erroring network client.
+  it('Check if account exists - failing network request w/ NOT_FOUND error', async function() {
+    // GIVEN a XpringClient which wraps an erroring network client that returns an Error containing
+    // a property 'code' with value grpc.status.NOT_FOUND.
+    const errorWithCode5 = new FakeServiceError(
+      'FakeServiceError: account not found',
+      grpc.status.NOT_FOUND,
+    )
+    const fakeNetworkClientResponses = new FakeNetworkClientResponses(
+      errorWithCode5, // getAccountInfoResponse
+    )
+    const fakeErroringNetworkClient = new FakeNetworkClient(
+      fakeNetworkClientResponses,
+    )
     const xpringClient = new DefaultXpringClient(fakeErroringNetworkClient)
 
-    // WHEN accountExists throws an exception while calling getBalance
+    // WHEN accountExists encounters this Error while calling getBalance
```

Comment:
Generally, encourage you to think of GIVEN, WHEN, THEN as a human reading it, rather than just writing what the test is doing.

You could make this as simple as:
// GIVEN A xpring client with a network client that will report accounts as not found
// WHEN Account existence is checked
// THEN The account is reported to exist

Reply:
this makes sense too - less reference to actual names of things, and more of an abstract conception of what's happening